### PR TITLE
transfer player to another team

### PR DESCRIPTION
**Pull request by FernandoArevaloE**
* allows changing the team where a player is assigned

### CODE REVIEW

File: web/src/views/PlayerList.js [hunk 1]
```diff
@@ -28,12 +54,15 @@ const PlayerList = () => {
             const headers = {
                 'Authorization': `Bearer ${token}`
             }
-            await axios.get(`http://localhost:4000/v1/team/${id}/player`,
+            axios.get(`http://localhost:4000/v1/team/${id}/player`,
```
Comment (andymotagl) [inline web/src/views/PlayerList.js:57]: What happens if you make the request have a delay?

File: web/src/views/PlayerList.js [hunk 2]
```diff
@@ -48,23 +77,102 @@ const PlayerList = () => {
...
+            'Authorization': `Bearer ${token}`
+        }
+        axios.get(`http://localhost:4000/v1/company/${companyId}/team`,
+            {
+                headers: headers
+            }
+        )
+        .then(response => {
+            let data = null;
+            if (response.data.data.length > 0) {
```
Comment (andymotagl) [inline web/src/views/PlayerList.js:102]: If I am not wrong, the response is 404 when the list is empty, then no need of doing this.

File: web/src/views/PlayerList.js [hunk 3]
```diff
@@ -87,16 +196,95 @@ const PlayerList = () => {
                             <tbody>
                                 {playerList.map(player => (
                                     <tr key={player.player_id}>
+                                        <th>
+                                            {player.active && <Input
```
Comment (andymotagl) [inline web/src/views/PlayerList.js:200]: Let the `<Input` starts in the line below

**Review by andymotagl — COMMENTED** [review]
Great job! Besides those small comments, everything looks great!

File: web/src/views/PlayerList.js (hunk 1 above)
Comment (FernandoArevaloE) [reply web/src/views/PlayerList.js:57]: what purpose with?

Comment (andymotagl) [reply web/src/views/PlayerList.js:57]: If the request takes a bunch of time, your component renderizes without waiting for it. Is that expected?

**Review by sebastian-motavita — APPROVED** [review]

**Pull request merged**